scalableminds/webknossos
# make decimal position input possible

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open a dataset
- paste a visibly different position with decimal places to the position input 
- the view should jump to that position
- on refocusing on the input field the decimal places should vanish

### Issues:
- fixes #4777

------
